modules/light-clients/08-wasm/keeper/genesis.go

```diff
@@ -10,7 +10,7 @@ import (
 // state.
 func (k Keeper) InitGenesis(ctx sdk.Context, gs types.GenesisState) error {
 	for _, contract := range gs.Contracts {
-		_, err := k.storeWasmCode(ctx, contract.CodeBytes)
+		_, err := k.storeWasmCode(ctx, contract.CodeBytes, k.wasmVM.StoreCodeUnchecked)
```

Comment:
note: these calls to `k.wasmVM.FuncCall` should be replaced with `ibcwasm.GetVM()` in another backport so that all are consistent